.eslintrc.json

```diff
@@ -6,6 +6,7 @@
         "window": false
     },
     "rules": {
-        "semi": ["error", "never"]
+        "semi": ["error", "never"],
+        "linebreak-style": 0
```

Comment:
why did you disable this rule?

Reply:
Because ESLint marks errors on all the lines.

Reply:
All the lines?...

Reply:
https://eslint.org/docs/rules/linebreak-style

Reply:
@Vector-Blanxart  The 'linebreak-style' rule works perfectly. check the configuration of your local.

![pxngg-wdc9s](https://user-images.githubusercontent.com/19965590/44273450-8980e300-a20d-11e8-8754-2ec7d7cf5950.gif)